dotnet/src/dotnetframework/Providers/Cache/GxRedis/GxRedis.cs

```diff
@@ -293,5 +412,89 @@ static T Deserialize<T>(string value)
 			opts.Converters.Add(new ObjectToInferredTypesConverter());
 			return JsonSerializer.Deserialize<T>(value, opts);
 		}
+#if NETCORE
+		private TimeSpan LocalCacheTTL(int durationMinutes)
+		{
+			return LocalCacheTTL(durationMinutes > 0 ? TimeSpan.FromMinutes(durationMinutes) : (TimeSpan?)null);
+		}
+		private TimeSpan LocalCacheTTL(TimeSpan? ttl)
+		{
+			return ttl.HasValue ? TimeSpan.FromTicks((long)(ttl.Value.Ticks * DEFAULT_LOCAL_CACHE_FACTOR)) : LOCAL_CACHE_PERSISTENT_KEY_TTL;
+		}
+#endif
+		private void ClearKeyLocal(string key)
+		{
+#if NETCORE
+			if (_localCache == null)
+				return;
+			_localCache.Remove(key);
+#endif
+		}
+		void ClearAllCachesLocal()
+		{
+#if NETCORE
+			if (_localCache == null)
+				return;
+			_localCache.Compact(1.0);
+#endif
+		}
+
+		private void KeyExpireLocal(string fullKey)
+		{
+#if NETCORE
+			if (_localCache == null)
+				return;
+			_localCache.Remove(fullKey);
+#endif
+		}
+		private bool KeyExistsLocal(string fullKey)
+		{
+#if NETCORE
+			if (_localCache == null)
+				return false;
+			if (_localCache.TryGetValue(fullKey, out _))
+				return true;
+#endif
+			return false;
+		}
+
+		private void SetLocal<T>(string key, T value)
+		{
+#if NETCORE
+			if (_localCache != null)
+			{
+				TimeSpan? redisTTL = RedisDatabase.KeyTimeToLive(key);
+				_localCache.Set(key, value, LocalCacheTTL(redisTTL));
+			}
+#endif
+		}
+		private void SetPersistentLocal(string cacheid, long? prefix)
+		{
+#if NETCORE
+			if (_localCache != null) 
+				_localCache.Set(cacheid, prefix, LocalCacheTTL(LOCAL_CACHE_PERSISTENT_KEY_TTL));
```

Comment:
You can use null conditional operator to combine the null check with the actual invocation. e.g.
`_localCache?.Set(cacheid, prefix, LocalCacheTTL(LOCAL_CACHE_PERSISTENT_KEY_TTL));`

Reply:
Similarly in other places it can be used as well in replacement of checking for non null, and then doing something with the field.